### DISCUSSION TITLE
error: Reference to FabricBlockSettings is ambiguous

### DISCUSSION THREAD

**Discussion by Atten007** [FabricMC/fabric-api#2022] [Mod Dev Support] [answered]
Hi folks,

I'm currently making a 2nd version of "ComputerCraft" with Mcreator for Minecraft fabric 1.16.5. But when Mcreator tries to compile some mod elements, a very shitty error named "Reference to FabricBlockSettings is ambiguous" occurs and also the shitty error "A type with the same simple name is already defined by the single-type-import of FabricBlockSettings. What I've made wrong here? How can I fix these both very shitty errors? I'm very sorry for swearing!

Sincerely yours,
Atten007😊

**Comment by modmuss50** [comment, marked as answer]
Without knowing what code its trying to compile its impossible to provide any help. I would report this to the Mcreator devs.

You may get more out of modding by writing the code yourself as it will provide you with the abbility to solve issues like this by your self.
